src/ActiveRoomsProvider.tsx

```diff
@@ -58,21 +58,21 @@ export const ActiveRoomsProvider = withChatkit(({ chatkit, children }: Props) =>
   }, []);
 
   const onAddedToRoom = (addedRoom: PusherRoom) => {
-    const sortedRooms = sortRooms([addedRoom, ...activeRooms]);
-    setActiveRooms(sortedRooms);
+    setActiveRooms(prev => sortRooms([addedRoom, ...prev]));
```

Comment:
can we use `prevActiveRooms` or `previousActiveRooms`? This functional updates are not so known